lib/flutter_tts.dart

```diff
@@ -183,6 +183,12 @@ class FlutterTts {
   Future<dynamic> setSharedInstance(bool sharedSession) async =>
       await _channel.invokeMethod('setSharedInstance', sharedSession);
 
+  /// [Future] which invokes the platform specific method for setting the autoStopSharedSession
+  /// default value is true
+  /// *** iOS, and macOS supported only***
```

Comment:
For macOS support, this would need to also be added to macos/Classes/FlutterTtsPlugin.swift

Reply:
I've added the handler but MacOS does not use an audio session so the handler just accepts the call